experimenter/tests/integration/nimbus/ios/test_ios_integration.py

```diff
@@ -0,0 +1,26 @@
+import time
+
+import pytest
+
+
+@pytest.mark.smoke
```

Comment:
Instead of `smoke` can we call this like `ios` or `ios_enrollment` or something?